substrate/frame/revive/src/benchmarking.rs

```diff
@@ -127,9 +127,7 @@ mod benchmarks {
 	// is not in the first basic block is never read. We are primarily interested in the
 	// `proof_size` result of this benchmark.
 	#[benchmark(pov_mode = Measured)]
-	fn call_with_code_per_byte(
-		c: Linear<0, { limits::code::STATIC_MEMORY_BYTES / limits::code::BYTES_PER_INSTRUCTION }>,
-	) -> Result<(), BenchmarkError> {
+	fn call_with_code_per_byte(c: Linear<0, { 100 * 1024 }>) -> Result<(), BenchmarkError> {
```

Comment:
why 100 * 1024 and not BLOB_BYTES?

Reply:
This gets really slow. And since it scales linearly we don't need to test with the biggest blob.